notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -47,7 +47,7 @@
                {% if fip_banner_french or brand_name == "canada.ca-fr" %}
 
                <img
-                src="https://notification-alpha-canada-ca-asset-upload.s3.amazonaws.com/09d0d711-9c23-48ec-b21e-fc6acd2412bf-gov-canada-fr.png"
+                src="https://assets.notification.canada.ca/gov-canada-fr.png"
```

Comment:
Copied the existing file with a clean name and keep the old filename as well.

https://assets.notification.canada.ca/09d0d711-9c23-48ec-b21e-fc6acd2412bf-gov-canada-fr.png